### DISCUSSION TITLE
template doesn't load , tryed more versions of kimai

### DISCUSSION THREAD

**Discussion by Freddy9393219** [kimai/kimai#4522] [On-Premise: Installation & Updates] [answered]
Hallo, i installed kimai a few times today (with ssh and plesk in my case) but allways it lokks like at my screenshot attached. what did I wrong?
![Screenshot 2024-01-02 160348](https://github.com/kimai/kimai/assets/155461772/6aece994-a108-4935-84d7-4a05eed484ad)

**Comment by Freddy9393219** [comment]
the url only was to testpurposes wrong set . it is set to kimai/public of cause

**Reply by kevinpapst** [reply]
KImai DOES NOT SUPPORT to be used in  a subdirectory. Point your document root to public/.
It is NOT allowed or supported that `public/` shows up in the URL.

I could shortcut that with RTM: 
important:https://www.kimai.org/documentation/installation.html#configure-documentroot

**Reply by Freddy9393219** [reply]
the root is at public . the screenshot was made before. i tested if it looks the same

**Reply by kevinpapst** [reply, marked as answer]
<img width="766" alt="Bildschirmfoto 2024-01-02 um 16 54 33" src="https://github.com/kimai/kimai/assets/533162/319d3cf2-d291-4051-af42-6bbd8f0ace8c">

I don't know what you are doing, the assets are not available. Those are installed automatically, when installing Kimai.
I am not going to tell you to run `bin/console assets:install` because this is automatically executed when you run composer as documented.

**Reply by Freddy9393219** [reply]
THX this was really the problem. I don't know why this happened?!

**Reply by kevinpapst** [reply]
Did you execute composer as documented or did you use a flag like `--no-scripts`?

**Reply by Freddy9393219** [reply]
as documented but i got the memory exeed error at rebuild after install (in the automatic script)
/opt/plesk/php/8.2/bin/php ../composer.phar install --optimize-autoloader

**Reply by kevinpapst** [reply]
Thank god, people read the docu: https://www.kimai.org/documentation/installation.html#memory-limit 😈
